app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -327,13 +315,29 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
         }
     }
 
+    private fun setEnginesDisabledWhileSyncing(isSyncing: Boolean) {
```

Comment:
There was weird behavior when you could interact with the toggles while syncing #6553 so I disabled them while syncing.